pkg/aws/blockindextablemapper.go

```diff
@@ -87,6 +93,17 @@ func (bit blockIndexTableMapper) GetClaims(ctx context.Context, contentHash mult
 				continue
 			}
 
+			// check if shard is a web3.storage shard
+			if strings.Join(parts[:2], "/") == "us-west-2/dotstorage-prod-1" {
```

Comment:
As per previous comment this needs to be altered to also look for `dotstorage-prod-0` in the east and discount the keys with `nft-` in them.